agent/ansible/pbench/agent/roles/pbench_agent_config/README.md

```diff
@@ -0,0 +1,61 @@
+pbench_agent_config
+===================
+
+This role installs a pbench-agent.cfg file and an ssh key. The former
+defines the server to be used for
+pbench-move-results/pbench-copy-results and the latter allows the
+communication to happen.
+
+Requirements
+------------
+
+Role Variables
+--------------
+These variables are *NOT* defined by default. They *have* to be
+defined in the user's inventory file:
+
+-pbench_config_url: the location where the config file is to be fetched from
+-pbench_key_url: the location where the key file is to be fetched from
+
+These are the variables that are defined by default and do not
+generally need to be modified:
+
+- pbench_agent_install_dir: the installation directory for the pbench agent (default: /opt/pbench-agent)
+- pbench_owner: the owner of all the files in the installation directory (default: pbench)
+- pbench_group: the group of all the files in the installation directory (default: pbench)
+- pbench_config_files: the list of config files to be installed (default '["pbench-agent.cfg"]')
+- pbench_config_dest: the location where the config file(s) are to be installed (default: {{ pbench_agent_install_dir }}/config)
+- pbench_key_dest: the location where the key file is to be installed (default: {{ pbench_agent_install_dir }})
+
+
+Dependencies
+------------
+This role depends on another role in this collection:
+
+- pbench_agent_files_install
+
+Example Playbook
+----------------
+
+    - name: install pbench-agent config and key files
```

Comment:
Seems like we should reference an example playbook that is in the git repo?

Reply:
See above

Reply:
Done